### DISCUSSION TITLE
Chrome、Safari 和 Firefox 三大瀏覽器攜手打造新基準 Speedometer 3

### DISCUSSION THREAD

**Discussion by williamyen0623** [1101-GS3073-C/Python-Community-GS3073-C#557] [科技新知]
https://www.inside.com.tw/article/30156-google-apple-mozilla-speedometer-3

相信大家都知道現在的三大瀏覽器，Chrome、Safari、Firefox，這些瀏覽器的母公司打算攜手打造優化的網頁瀏覽器基準 Speedometer 3，新基準將用以提升瀏覽器的效率
我自己是三個瀏覽器都有在交錯使用，各有好壞，但我蠻期待這次的計畫的，真令人興奮！

**Comment by HsuChen88** [comment]
瀏覽器各方面的統一對網頁開發者來說也是一件好事，能提高程式在各瀏覽器上執行的穩定性。

**Comment by 0212Jerry** [comment]
舊一代的Ie瀏覽器消失，現在要推出新的出來，個人十分期待。我自己是比較常用Google跟safari。就使用上我更喜歡google 的流暢度以及搜尋結果。

**Comment by Vincent20030314** [comment]
瀏覽器是一項我每天ㄉ都會用的工作，真心期待它整合過的效能。
